src/covid_model_seiir_pipeline/postprocessing/model/loaders.py

```diff
@@ -0,0 +1,125 @@
+import functools
```

Comment:
Moved all the logic in these functions to the data interface.  Now this is a wrapper to support concurrent reads.